File: payroll-api/src/main/java/com/company/payroll/mapper/StaffSalaryMapper.java
package com.company.payroll.mapper;

import java.util.List;

import com.company.payroll.model.StaffSalary;
import org.springframework.stereotype.Repository;

@Repository
public interface StaffSalaryMapper {
    int deleteByPrimaryKey(Integer sId);
    int insert(StaffSalary row);
    int insertSelective(StaffSalary row);
    List<StaffSalary> selectList();
    StaffSalary selectByPrimaryKey(Integer sId);
    int updateByPrimaryKeySelective(StaffSalary row);
    int updateByPrimaryKey(StaffSalary row);
}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/mapper/StaffSalaryMapper.kt
package com.company.payroll.mapper

import com.company.payroll.model.StaffSalary
import org.springframework.stereotype.Repository

@Repository
interface StaffSalaryMapper {
  fun deleteByPrimaryKey(sId: Int): Int

  fun insert(row: StaffSalary): Int

  fun insertSelective(row: StaffSalary): Int

  fun selectByPrimaryKey(sId: Int): StaffSalary?

  fun updateByPrimaryKeySelective(row: StaffSalary): Int

  fun updateByPrimaryKey(row: StaffSalary): Int
}